js/packages/glimmer-idc/src/components/advanced-search/search-tips.ts

```diff
@@ -16,10 +16,18 @@ export default class SearchTips extends Component<Args> {
           <ul class="list-disc list-inside">
             <li class="my-1">Use quotation marks to search as a phrase</li>
             <li class="my-1">
-              Use the <span class="text-lg font-bold">%</span> wildcard character to search for words with one alternate character
+              Use the <span class="text-lg font-bold">?</span> wildcard character to search for words with one alternate character. For example, <span class="italic">te?t</span> should match test and text
             </li>
             <li class="my-1">
-              Use the <span class="text-lg font-bold">?</span> wildcard character to search for words with multiple alternate characters
+              Use the <span class="text-lg font-bold">*</span> wildcard character to search for words with multiple alternate characters. Searching for <span class="italic">test*</span> should match test, tester, testing, etc
+            </li>
+            <li class="my-1">
+              Use the proximity search syntax if you want to search for two terms within a certain number of words of each other. The term <span class="italic">"farm goat"~10</span>,
```

Comment:
I wonder if people will be confused and think that they need to use the operator described here as opposed to using the UI constructed for this (with the inputs, etc)